src/dmd/parse.d

```diff
@@ -3222,20 +3222,24 @@ final class Parser(AST) : Lexer
                         error("if type, there must be an initializer");
                 }
 
-                AST.UserAttributeDeclaration uad;
-                if (udas)
-                    uad = new AST.UserAttributeDeclaration(udas, null);
-
                 AST.DeprecatedDeclaration dd;
                 if (deprecationMessage)
                 {
                     dd = new AST.DeprecatedDeclaration(deprecationMessage, null);
                     stc |= STC.deprecated_;
                 }
 
-                auto em = new AST.EnumMember(loc, ident, value, type, stc, uad, dd);
+                auto em = new AST.EnumMember(loc, ident, value, type, stc, null, dd);
                 e.members.push(em);
 
+                if (udas)
+                {
+                    auto s = new AST.Dsymbols();
+                    s.push(em);
```

Comment:
Looks like this introduced a regression (https://github.com/dlang/dmd/pull/14522), was this addition needed for something?

Reply:
I don't remember exactly, I think it was just for completeness, the Uda holding the symbol and the symbol holding the uda.
If it works without it, just remove it. `syntaxCopy/arraySyntaxCopy` recursion bugs suck.